package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.1.4",
   "description": "The JS library containing the Monero crypto plus lightweight wallet functions behind the official MyMonero apps",
   "main": "index.js",
+  "react-native": "index.rn.js",
```

Comment:
This only adds a new entry for React Native. Other targets continue using existing code.